hw/ip/spi_device/data/spi_device.hjson

```diff
@@ -687,10 +724,31 @@
               this case, SW sets payload_en to 4'b 0010.
               '''
           }
-          { bits:   "20"
-            name:   "payload_dir"
-            desc:   "Set to 1 if the command returns data. If 0, the payload sends to the downstream Flash device."
-          }
+          { bits: "20"
```

Comment:
just to refresh my memory.. the `cmd_info` registers allow you to define specific attributes about an op-code right?
so this is basically saying..if it matches the op-code, we can optionally enable the payload swap? This would allow us to do the write-status payload swap in a generic way, is that right?

Reply:
Right. So, SW may configure one of available cmd_info slot (index 11 to 23) and set valid bit, opcode to 0x01, set `payload_swap_en` and configure PAYLOAD_SWAP_MASK and PAYLOAD_SWAP_DATA.